pkg/scalers/couchdb_scaler.go

```diff
@@ -176,9 +176,8 @@ func parseCouchDBMetadata(config *ScalerConfig) (*couchDBMetadata, string, error
 		// nosemgrep: db-connection-string
 		connStr = "http://" + addr
 	}
-	if val, ok := config.TriggerMetadata["metricName"]; ok {
```

Comment:
This shouldn't be removed now, just made optional as it is for other scalers.

Reply:
@yardenshoham I've just found this nit^

Reply:
I removed it because it does nothing as far as I can tell. `meta.metricName` is overidden on the next line

Reply:
you are right, probably some leftover, the parameter is not mentioned in the docs 🤷‍♂️ so we can skip this scaler. Thanks!